### PR TITLE
Add the pod resource to the test controller

### LINKED ISSUE #779: Use Pods as Test Data in Mock Controllers
The current resources available for seeding test data for mock controllers is shown [here](https://github.com/tektoncd/triggers/blob/45daf11629f2709c359b5d6a3d5fad37939e96ec/test/controller.go#L55). I am wondering if it would make sense to add Pods to this collection of resources to aid with unit testing EventListener logs for `tkn` amongst other testing use cases. Or is there an alternative way to make use of `SeedResources` to include Pods?

/kind question
/kind feature

### PR DESCRIPTION
**Pull request by khrm**
Add the pod resource to the test controller which is used by tkn cli to test el pod.
Fixes #779
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes
```release-note
NONE
```

### CODE REVIEW

File: vendor/knative.dev/pkg/client/injection/kube/informers/core/v1/pod/fake/fake.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/*
```
Comment (savitaashture) [inline vendor/knative.dev/pkg/client/injection/kube/informers/core/v1/pod/fake/fake.go:1]: how this file added? 
is there any changes in go.mod file

Comment (khrm) [reply vendor/knative.dev/pkg/client/injection/kube/informers/core/v1/pod/fake/fake.go:1]: Its package is already under go.mod.  Changes are in vendor/modules.txt file.

Comment (dibyom) [reply vendor/knative.dev/pkg/client/injection/kube/informers/core/v1/pod/fake/fake.go:1]: Its due to the new import here: 	fakepodinformer "knative.dev/pkg/client/injection/kube/informers/core/v1/pod/fake"

**Pull request merged**